src/main/java/frc/robot/commands/leftIntakeGoTo.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.Constants;
+
+public class leftIntakeGoTo extends CommandBase {
```

Comment:
Classes (and Command) names should start with a capital letter to differentiate them from variables.  This should be LeftIntakeGoTo().